index.html

```diff
@@ -3270,38 +3270,17 @@ <h2 id="incubations">
       </h2>
       <p>
         Extensions to this specification are being incubated in parallel by the
-        Web Community, some of which are shipping in multiple browsers. If two
-        or more browser engines end up supporting an incubated feature, then
-        those features will become part of this specification in the future -
-        allowing them to become a standard the Web Platform:
+        Web Community, some of which are shipping in multiple browsers. All
+        incubations can be found in the <a href=
+        "https://github.com/WICG/manifest-incubations">Manifest Incubations
```

Comment:
I think this isn't quite true, we have a number of repositories, and this one hosts a bunch of stuff, but we also have:
- https://wicg.github.io/web-app-launch/
- https://wicg.github.io/web-share-target/
- maybe more? We can just list them though.

Reply:
I think it would be better if we could link to Manifest Incubations as the single document that refers to all incubations. This way, we can avoid noise in this spec.

The [`share_target` member](https://wicg.github.io/manifest-incubations/#share_target-member) is already listed in Manifest Incubations and links to the Share Target spec. The Web App Launch Handler API is linked in the references section, but it seems like the document doesn’t use the reference right now. We could add the `launch_handler` member and link to its spec in the same way.

WDYT? @dmurph @marcoscaceres